smithy-model/src/main/java/software/amazon/smithy/model/validation/validators/EnumShapeValidator.java

```diff
@@ -23,6 +23,8 @@
 import java.util.regex.Pattern;
```

Comment:
Added validation here that was previoulsy only performed in the IDL and not in the AST loader.